lisa/sut_orchestrator/qemu/schema.py

```diff
@@ -0,0 +1,31 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from dataclasses_json import dataclass_json
+
+
+# Configuration options for cloud-init ISO generation for the VM.
+@dataclass_json()
+@dataclass
+class CloudInitSchema:
+    # Additional values to apply to the cloud-init user-data file.
+    extra_user_data: Optional[str] = None
+
+
+# QEMU orchestrator's global configuration options.
+@dataclass_json()
+@dataclass
+class QemuPlatformSchema:
+    pass
+
+
+# QEMU orchestrator's per-node configuration options.
+@dataclass_json()
+@dataclass
+class QemuNodeSchema:
```

Comment:
@cwize1 We already have a Qemu tool that can spin up a nested VM on Azure or ready platform. Can you please clarify the additional functionalities this platform will add?

Reply:
I believe that tool is for testing nested VM functionality. Whereas this orchestrator is for running the LISA test framework on a Linux (KVM/QEMU) box.

Reply:
Got it! Thanks for clarification.